flake.nix

```diff
@@ -31,19 +31,14 @@
   in {
     defaultBundler = builtins.listToAttrs (map (system: {
         name = system;
-        value = drv: self.bundlers.${system}.toArx (protect drv);
-      }) supportedSystems)
-      # Backwards compatibility helper for pre Nix2.6 bundler API
-      // {__functor = s: nix-bundle.bundlers.nix-bundle;};
+        value = drv: self.bundlers.${system}.default (protect drv);
+      }) supportedSystems);
+
+    bundlers =
+      (forAllSystems (system: rec {
 
-    bundlers = let n =
-      (forAllSystems (system: {
-        # Backwards compatibility helper for pre Nix2.6 bundler API
-        toArx = drv: (nix-bundle.bundlers.nix-bundle ({
-          program = if drv?program then drv.program else (program drv);
-          inherit system;
-        })) // (if drv?program then {} else {name=
-          (builtins.parseDrvName drv.name).name;});
```

Comment:
Because of the removal of this we're now using https://github.com/nix-community/nix-bundle/blob/59fbe778bd07e317e4fa4b667510b0129a6f0727/default.nix#L25 as the symlink name
The symlink target remains the same.